src/components/loginRedirect/LoginRedirect.jsx

```diff
@@ -0,0 +1,238 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import { BackHandler, View } from 'react-native'
+import GooddollarImage from '../../assets/gooddollarLogin.svg'
+
+import { CustomButton, Section, Text } from '../common'
+import useProfile from '../../lib/userStorage/useProfile'
+import API from '../../lib/API/api'
+import goodWallet from '../../lib/wallet/GoodWallet'
```

Comment:
should be replaced with useWallet hook (example)

```javascript
import { useWallet } from '../../lib/wallet/GoodWalletProvider'

// inside component
const goodWallet = useWallet()
```

Reply:
to get it, also merge latest master to Your branch

Reply:
done